azurerm/internal/services/monitor/tests/monitor_action_rule_action_group_resource_test.go

```diff
@@ -0,0 +1,252 @@
+			return err
+		}
+		if resp, err := client.GetByName(ctx, id.ResourceGroup, id.Name); err != nil {
+			if !utils.ResponseWasNotFound(resp.Response) {
+				return fmt.Errorf("bad: Get on Monitor ActionRulesClient: %+v", err)
+			}
+		}
+		return nil
+	}
+	return nil
+}
+
+func testAccAzureRMMonitorActionRuleActionGroup_basic(data acceptance.TestData) string {
+	template := testAccAzureRMMonitorActionRuleActionGroup_template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_monitor_action_rule_action_group" "test" {
+  name                = "acctest-moniter-%d"
+  resource_group_name = azurerm_resource_group.test.name
+  action_group_id     = azurerm_monitor_action_group.test.id
+}
+`, template, data.RandomInteger)
+}
+
+func testAccAzureRMMonitorActionRuleActionGroup_requiresImport(data acceptance.TestData) string {
+	template := testAccAzureRMMonitorActionRuleActionGroup_basic(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_monitor_action_rule_action_group" "import" {
+  name                = azurerm_monitor_action_rule_action_group.test.name
+  resource_group_name = azurerm_monitor_action_rule_action_group.test.resource_group_name
+  action_group_id     = azurerm_monitor_action_rule_action_group.test.action_group_id
+}
+`, template)
+}
+
+func testAccAzureRMMonitorActionRuleActionGroup_complete(data acceptance.TestData) string {
+	template := testAccAzureRMMonitorActionRuleActionGroup_template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_monitor_action_rule_action_group" "test" {
+  name                = "acctest-moniter-%d"
+  resource_group_name = azurerm_resource_group.test.name
+  action_group_id     = azurerm_monitor_action_group.test.id
+  enabled             = false
+  description         = "actionRule-test"
+
+  scope {
+    type         = "ResourceGroup"
+    resource_ids = [azurerm_resource_group.test.id]
+  }
+
+  condition {
+    alert_context {
+      operator = "Contains"
+      values   = ["context1", "context2"]
+    }
+
+    alert_rule_id {
+      operator = "Contains"
+      values   = ["ruleId1", "ruleId2"]
+    }
+
+    description {
+      operator = "DoesNotContain"
+      values   = ["description1", "description2"]
+    }
+
+    monitor {
+      operator = "NotEquals"
+      values   = ["Fired"]
+    }
+
+    monitor_service {
+      operator = "Equals"
+      values   = ["Data Box Edge", "Data Box Gateway", "Resource Health"]
+    }
+
+    severity {
+      operator = "Equals"
+      values   = ["Sev0", "Sev1", "Sev2"]
+    }
+
+    target_resource_type {
+      operator = "Equals"
+      values   = ["Microsoft.Compute/VirtualMachines", "microsoft.batch/batchaccounts"]
+    }
+  }
+
+  tags = {
+    ENV = "Test"
+  }
+}
+`, template, data.RandomInteger)
+}
+
+func testAccAzureRMMonitorActionRuleActionGroup_template(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctest-rg-%d"
```

Comment:
for monitor resources
```suggestion
  name     = "acctest-monitor-%d"
```